deploy/helm/rawfile-localpv/values.yaml

```diff
@@ -129,6 +129,9 @@ node:
   # -- Data dir path for provisioner to be used by provisioner
   dataDirPath: /var/csi/rawfile
 
+  # -- Kubelet path (Set to `/var/lib/k0s/kubelet` for k0s) without trailing slash
+  kubeletPath: /var/lib/kubelet
```

Comment:
I suggest we add a template function to generate a `kubeletPath` without the suffix and use that instead of the var directly

Reply:
Can you give me an example from other local PVs, I mean we are depending on the real path of the kubelet in some places